rn/Teacher/src/canvas-api-v2/queries/ContextCard.js

```diff
@@ -70,7 +70,6 @@ export const courseQuery = gql`query StudentContextCardCourse($courseID: ID!, $u
                 late
                 missing
                 onTime
-                total
```

Comment:
hmm, looking at the object maybe this total includes `missing`, so that's probably what's up.

Reply:
Yeah, that would make sense, but it returns "0, 0, 0, 6". Why? Who knows...